.spelling

```diff
@@ -77,6 +77,7 @@ init
 labeled
  - demos/Azure/README.md
 alpha.11
```

Comment:
`> get-childitem .\.spelling | select-string "alpha."
.spelling:79:alpha.11
.spelling:80:alpha.12
.spelling:93:alpha.8
.spelling:201:alpha.9
.spelling:258:alpha.7`
There are a bunch of others as well